kotlin/src/main/kotlin/org/jetbrains/research/testspark/kotlin/KotlinPsiMethodWrapper.kt

```diff
@@ -68,6 +68,22 @@ class KotlinPsiMethodWrapper(val psiFunction: KtFunction) : PsiMethodWrapper {
         return lineNumber in startLine..endLine
     }
 
+    fun getInterestingPsiClassesWithQualifiedNames(): MutableSet<PsiClassWrapper> {
+        val interestingPsiClasses = mutableSetOf<PsiClassWrapper>()
+
```

Comment:
As I understood, here we get the parent classes that are non-standard Kotlin classes. Could you write a Javadoc comment on this method?